### DISCUSSION TITLE
Is possible to use netlifyCMS?

### DISCUSSION THREAD

**Discussion by ivanhertanto** [thuliteio/doks#549] [Q&A] [answered]
I use this theme its great, anyway how to connect with netlifyCMS? it's posible? any tutorial or something like that 

thx before

i test it in doks.ivan.click

**Comment by h-enk** [comment, marked as answer]
I'm working on a one command setup for a headless CMS for Doks. Netlify CMS was on my short list, but didn't make it (not stable enough, lacks a number of functionalities, and is not that user friendly — in my opinion — amongst others). So, I would (strongly) advice against using Netlify CMS with Doks.

The winner (from my shortlist) for my headless CMS setup for Doks was (clearly) [Forestry](https://forestry.io/) (which will later become [Tina](https://tina.io/)). Since it will be a one-command-install solution which will work without manual configuration, and will contain a number of automations, I'm planning to charge ( a little amount) for this convenience. But of course, you could always do it yourself, and  manually set things up, and configure Forestry yourself.

Not sure yet when Doks CMS will be available, but expect it to be released in approximately two weeks time.

**Reply by ivanhertanto** [reply]
Oh okay, thx for answering :)

**Reply by ImaCrea** [reply]
wow thanks. can't wait to try that out :)

**Reply by ImaCrea** [reply]
Hi @h-enk I was wondering how it's going on that side and if a CMS is still on the roadmap please? :)

**Reply by h-enk** [reply]
It's still on the roadmap (currently, almost beta ready). Not sure though when it will be available — I'm working on it /it's one of my main priorities now (but I need to finish some other things first). Will keep you posted.

**Reply by ImaCrea** [reply]
Very very very cool news @h-enk :) You can count on me to test the beta ;)

**Reply by h-enk** [reply]
Hahaha, thanks!

**Reply by filipviz** [reply]
Thank you @h-enk, deeply appreciate your work! Any updates on a CMS system for Doks?

**Reply by ImaCrea** [reply]
Hello @h-enk hope you're going well :) Maybe some news on this side please? 🤗

**Reply by h-enk** [reply]
Hello, thanks for asking. I've got a lot (too much actually) going on (but, hey, what's new 😸). My first priority is releasing the new restructured Hyas — almost, almost, almost there. It's the new base for Doks and the CMS (plugins then). I also expect that things will get released faster then.

**Reply by ImaCrea** [reply]
wow 😱 Exciting news @h-enk 🙌
